internal_metrics/dimensions.yaml

```diff
@@ -0,0 +1,171 @@
+az:
+  brief: availability zone
+  description: availability zone
+  associated_metrics: ebpf_net.time_since_last_message_ns, ebpf_net.pipeline_message_error, ebpf_net.message, ebpf_net.entrypoint_info, ebpf_net.collector_log_count, ebpf_net.client_handle_pool, ebpf_net.client_handle_pool_fraction, ebpf_net.clock_offset_ns, ebpf_net.codetiming_avg_ns, ebpf_net.collector_health
+  example: us-east-1a
+
+c_host:
+  brief: Client host machine name.
+  description: Collector host machine name. This is a span or state that is reported by collector to reducer.
+  associated_metrics: ebpf_net.time_since_last_message_ns, ebpf_net.pipeline_message_error, ebpf_net.message, ebpf_net.entrypoint_info, ebpf_net.collector_log_count, ebpf_net.client_handle_pool, ebpf_net.client_handle_pool_fraction, ebpf_net.clock_offset_ns, ebpf_net.collector_health
+  example: ip-192-168-110-244.ec2.internal
+
+c_type:
+  brief: Client type
+  description: Client types are numbers desginating different client types. Different types are kernel(1), cloud(2), k8s(3), ingest(4), matching(5), aggregation(6), liveness_probe (7), readiness_probe(8).
+  associated_metrics: ebpf_net.time_since_last_message_ns, ebpf_net.pipeline_message_error, ebpf_net.message, ebpf_net.entrypoint_info, ebpf_net.collector_log_count, ebpf_net.client_handle_pool, ebpf_net.client_handle_pool_fraction, ebpf_net.clock_offset_ns, ebpf_net.collector_health
+  example: 1
+
+cloud:
+  brief: Cloud type
+  description: Cloud provider type where network explorer is installed.
+  associated_metrics: ebpf_net.time_since_last_message_ns, ebpf_net.pipeline_message_error, ebpf_net.message, ebpf_net.collector_log_count, ebpf_net.client_handle_pool, ebpf_net.client_handle_pool_fraction, ebpf_net.collector_health
+  example: 1
+
+detail:
+  brief: Detail
+  description: Detail status of a metric.
+  associated_metrics: ebpf_net.collector_health
+  example: 0
+
+env:
+  brief: environment
+  description: environment where network explorer was installed.
+  associated_metrics: ebpf_net.time_since_last_message_ns, ebpf_net.pipeline_message_error, ebpf_net.message, ebpf_net.entrypoint_info, ebpf_net.collector_log_count, ebpf_net.bpf_log, ebpf_net.client_handle_pool, ebpf_net.client_handle_pool_fraction, ebpf_net.collector_health
+  example: network-explorer-staging.
+
+error:
+  brief: error type
+  description: error type
+  associated_metrics: ebpf_net.pipeline_message_error, ebpf_net.entrypoint_info
+  example: fetched/unknown
+
+field:
```

Comment:
Depending on how detailed we decide we want to document some of the internal metrics, maybe remove dimensions from docs that are specific to bpf_log?

Reply:
This we can take to the team meeting tomorrow.